ccp/modules/teiler-ui-setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [ "$ENABLE_TEILER" == true ];then
```

Comment:
Where is this variable defined?

Reply:
This should be defined in the ccp.conf file